OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -136,7 +136,7 @@ internal static class DreamProcNativeRoot {
             if (!ascii.TryGetValueAsInteger(out int asciiValue))
                 throw new Exception($"{ascii} is not a number");
 
-            return new DreamValue(Convert.ToChar(asciiValue).ToString());
+            return new DreamValue(char.ConvertFromUtf32(asciiValue).ToString());
```

Comment:
The `ToString()` is unnecessary
```suggestion
            return new DreamValue(char.ConvertFromUtf32(asciiValue));
```